op-deployer/pkg/deployer/artifacts/cmd/mktar/main.go

```diff
@@ -0,0 +1,164 @@
+package main
+
+import (
+	"archive/tar"
+	"flag"
+	"io"
+	"log"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/klauspost/compress/zstd"
+)
+
+var (
+	baseDir = flag.String("base", "", "directory to archive")
+	outFile = flag.String("out", "", "path to output tzst")
+)
+
+func main() {
```

Comment:
Would be good to add some documentation (via small comment block) explaining what this program does. I'm curious what made you go with the golang program approach instead of via bash commands in the justfile as originally implemented. Maybe the tar cli command didn't provide enough configurability?